tutorials/eth-api/how-to-build-a-dapp.md

```diff
@@ -95,7 +95,7 @@ Before we start writing the smart contract, let's add a JSON-RPC URL to the conf
 ```javascript
 require('@nomicfoundation/hardhat-ethers');
 module.exports = {
-  solidity: '0.8.17',
+  solidity: '0.8.20',
```

Comment:
I went through this whole tutorial and caught some unrelated errors that are minor, can you please update them so the tutorial works as expected?

On line 678 and 742, the import should be:

```
import { useContractFunction, useEthers, MoonbaseAlpha } from '@usedapp/core';
```

On line 711 and 750, the `const` should include `chainId` and switchNetwork`:

```
  const { account, chainId, switchNetwork } = useEthers();
```